config/rbac/kustomization.yaml

```diff
@@ -7,3 +7,5 @@ resources:
 - service_account.yaml
 - role.yaml
 - role_binding.yaml
+- net_attach_def_role.yaml
```

Comment:
I would call it `net_attach_def_editor_role`

Reply:
Done